Source/Extensions/Blazorise.DataGrid/DataGridSelectColumn.cs

```diff
@@ -72,6 +72,26 @@ public override string FormatDisplayValue( object value )
     /// </summary>
     [Parameter] public bool DefaultItemHidden { get; set; } = false;
 
+    /// <summary>
+    /// Display text of the default item shown in filter dropdowns.
+    /// </summary>
+    [Parameter] public string FilterDefaultItemText { get; set; }
+
+    /// <summary>
+    /// Value of the default item used in filter dropdowns.
+    /// </summary>
+    [Parameter] public object FilterDefaultItemValue { get; set; }
+
+    /// <summary>
+    /// If true, disables the default item in filter dropdowns.
+    /// </summary>
+    [Parameter] public bool FilterDefaultItemDisabled { get; set; } = false;
```

Comment:
I think `= false;` is not necesary. Not sure why we have it for the above parameters too. 🤔